streams/in.m3u

```diff
@@ -702,8 +726,14 @@ https://ythls.onrender.com/channel/UCDCMjD1XIAsCZsYHNMGVcog.m3u8
 https://6n3yope4d9ok-hls-live.5centscdn.com/vaanavil/TV.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="VanithaTV.in",Vanitha TV (720p) [Not 24/7]
 https://ythls.onrender.com/channel/UCTcH-GOkexQHWKBWSfOjhPQ.m3u8
+#EXTINF:-1 tvg-id="VBCNews.in",VBC News (576p)
+https://5a1178b42cc03.streamlock.net/8200/8200/playlist.m3u8
+#EXTINF:-1 tvg-id="VCV.in",VCV (720p)
+https://5a1178b42cc03.streamlock.net/8210/8210/playlist.m3u8
```

Comment:
Not loading for me. Either Not 24/7 or Geo-Blocked.

Reply:
Accessible using VPN. So, not Geo-Blocked. 
![image](https://github.com/iptv-org/iptv/assets/61133303/ba257d7a-f71d-4cd7-9208-89f8cfd1fbb1)